.changeset/metal-impalas-allow.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/techdocs-common': patch
+---
+
+1. Techdocs publishers constructors now use parameter objects when being instantiated
+
+2. The `LocalPublish` publisher can now be created using `fromConfig`:
```

Comment:
Perhaps worth noting that this is an internal refactor only, because it is right? :grin:

Could perhaps even slip the changeset down a bit and just state that there was an internal refactor to align how publishers are created

Reply:
@Rugvip I've added a note that this is an internal refactor that should not impact the usage of the local publisher.

I didn't quite follow the request to, "slip the changeset down a bit", please let me know if there are any other changes you will like. Thanks!

Reply:
slim* xP, all good to

Reply:
Ahh! Gotcha.